examples/handlebars_templates/src/main.rs

```diff
@@ -46,6 +47,27 @@ fn rocket() -> rocket::Rocket {
         .catch(errors![not_found])
 }
 
+// when you want to register handlebars helpers
+#[allow(dead_code)]
+fn rocket_with_hbs_helper() -> rocket::Rocket {
+    rocket::ignite()
+        .mount("/", routes![index, get])
+        .attach(Template::config_and_fairing(|ctxt| {
+            ctxt.engines_mut()
+                .handlebars()
+                .register_helper("test",
```

Comment:
We need a real test here. Create a helper than does something, and use it in one of the templates. Ensure that the returned value is what's expected.